example_commands.md

```diff
@@ -11,7 +11,7 @@ docker run --rm --net cbio-net \
     -v "$PWD"/study-dir:/study:ro \
     -v "$HOME"/Desktop:/outdir \
     cbioportal-image \
-    metaImport.py -u http://cbioportal-container:8080/cbioportal -s /study --html=/outdir/report.html
+    sh -c 'exec metaImport.py --jar_path=/usr/local/tomcat/webapps/cbioportal/WEB-INF/lib/core-*.jar -u http://cbioportal-container:8080/cbioportal -s /study --html=/outdir/report.html'
```

Comment:
Unless this breaks other parts of the system that depend on `$PORTAL_HOME` referring to the actual source.